packages/@biomejs/tailwindcss-config-analyzer/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "tailwindcss-config-analyzer",
+  "version": "1.0.0",
+  "description": "Tools to introspect Tailwind CSS configurations for integration with Biome",
+  "private": true,
+	"license": "MIT OR Apache-2.0",
+  "author": "Dani Guardiola",
+  "type": "module",
+  "scripts": {
+    "gen:tailwind-preset": "bun src/generate-tailwind-preset.ts"
+  },
+  "dependencies": {
+    "tailwindcss": "^3.4.1"
+  },
+  "devDependencies": {
+    "@types/bun": "^1.0.6",
+    "bun": "^1.0.28",
```

Comment:
Let's not add Bun as a dependency

Reply:
Bun is used as a runner for the codegen step, as a simpler alternative to others like tsx or ts-node. I think it's fine.